src/Products/EasyNewsletter/views/newsletter_issue_send.py

```diff
@@ -233,13 +234,25 @@ def _get_recipients(self):
 
                 enl_receivers.append(enl_receiver)
 
+        receivers_raw = enl_receivers
+
         # get subscribers over selected plone members anpid groups
-        # plone_receivers = self.context.plone_subscribers
+        plone_receivers = []
+        try:
+            plone_receivers_adapter = IPloneUserGroupRecipients(self.context)
+        except TypeError:
+            plone_receivers_adapter = None
+        if not plone_receivers_adapter:
+            try:
+                plone_receivers_adapter = IPloneUserGroupRecipients(enl)
+            except TypeError:
+                plone_receivers_adapter = None
+        if plone_receivers_adapter:
+            plone_receivers = plone_receivers_adapter.get_plone_subscribers()
```

Comment:
maybe we should rename this to get_plone_receivers to make it more clear that these are not subscribers but receivers collected from plone members/groups.

Reply:
Yeah sounds good